src/Backend/Modules/Tags/Actions/Edit.php

```diff
@@ -148,13 +149,13 @@ private function validateForm()
 
             // no errors?
             if ($this->frm->isCorrect()) {
-                // build tag
-                $item['id'] = $this->id;
-                $item['tag'] = $this->frm->getField('name')->getValue();
-                $item['url'] = BackendTagsModel::getURL(
-                    CommonUri::getUrl(\SpoonFilter::htmlspecialcharsDecode($item['tag'])),
+                $item = $this->record;
```

Comment:
Why this line? Can't you just do do all operations on $this->record? I feel no need for taking a copy of this variable.